core/templates/dev/head/pages/state_editor/StateSolutionEditorDirective.js

```diff
@@ -57,6 +57,16 @@ oppia.directive('stateSolutionEditor', [
           $scope.StateInteractionIdService = StateInteractionIdService;
           $scope.StateSolutionService = StateSolutionService;
 
+          $scope.getTooltip = function() {
```

Comment:
Try and be clearer in the name about what sort of tooltip this is / what the tooltip is for / what it is attached to. "getTooltip" alone is very generic...

Reply:
Done